### DISCUSSION TITLE
Allow PowerToys Run shortcut (Alt+D) to work with Glaze Window Manager

### DISCUSSION THREAD

**Discussion by chanithecat** [glzr-io/glazewm#633] [Q&A]
I'm using Glaze Window Manager alongside PowerToys on Windows, and I've encountered an issue with keyboard shortcuts. Specifically, I'm trying to use Powertoys Run witch I mapped to Alt+D shortcut to open files quickly.

But it seems that Glaze might be blocking or intercepting keyboard shortcuts that arent explicitly mapped in its configuration. This prevents me from using powertoys run effectively.

My questions are:

1. Is this a known issue with Glaze?
2. Is there a way to allow specific external shortcuts (like Alt+D for powertoys run) to work alongside Glaze?
3. If not currently possible, would you consider adding a feature to allow users to specify certain shortcuts that should be passed through to other applications?

Ive been loving Glaze's functionality but I'm not used to window's default search (Win+S) and I prefer powertoys run since thats what ive been using.

Thank you so much for your time and consideration.

**Comment by chanithecat** [comment]
I changed my Powertoys Run's shortcut to `ctrl+space` for now.

**Reply by k88936** [reply]
alt+d is in used by default: [see](https://github.com/glzr-io/glazewm/blob/151136657bff35800d70a2337c98ebe023532aca/resources/assets/sample-config.yaml#L263-L264),
I dont need it, so I commented this in my config
